### PR TITLE
Updated docs for Xbox GA

### PR DESCRIPTION
**Pull request by nishitha-burman**
Rendered article sections for review:
* Introduction to Microsoft Edge WebView2 > Supported platforms
   https://review.learn.microsoft.com/microsoft-edge/webview2/?branch=pr-en-us-2897#supported-platforms
* Get started with WebView2 in WinUI 2 (UWP) apps
   https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2?branch=pr-en-us-2897
* WebView2 Roadmap > GA > Xbox
   https://review.learn.microsoft.com/microsoft-edge/webview2/roadmap?branch=pr-en-us-2897#xbox

Updated the WebView2 Intro & Roadmap articles to mention Xbox & HoloLens 2 support.

AB#47285310

### CODE REVIEW

File: microsoft-edge/webview2/index.md [hunk 1]
```diff
@@ -88,6 +88,9 @@ WebView2 apps can run on the following versions of Windows:
 *  Windows Server 2019
 *  Windows Server 2016
 
+WebView2 is supported in the following devices:
```
Comment (mikehoffms) [inline microsoft-edge/webview2/index.md:91]:
```suggestion
WebView2 is supported on the following devices:
```

File: microsoft-edge/webview2/index.md [hunk 2]
```diff
@@ -88,6 +88,9 @@ WebView2 apps can run on the following versions of Windows:
...
+*  Xbox
```
Comment (mikehoffms) [inline microsoft-edge/webview2/index.md:92]:
```suggestion
*  Windows
*  Xbox
```
tech review this change

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Writer/Editor review

File: microsoft-edge/webview2/get-started/winui2.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ ms.date: 02/21/2023
 # Get started with WebView2 in WinUI 2 (UWP) apps
 
 In this tutorial, you:
-*  Set up your development tools for creating UWP apps that use WebView2 to display web content.
+*  Set up your development tools for creating UWP apps that use WebView2 to display web content. These instructions are also applicable to Xbox.
```
Comment (champnic) [inline microsoft-edge/webview2/get-started/winui2.md:14]: Should this be a separate note? Or a separate bullet point in this list?

**Review by champnic — APPROVED** [review]

File: microsoft-edge/webview2/get-started/winui2.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/get-started/winui2.md:14]: rendered article, intro at top:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2?branch=pr-en-us-2897

File: microsoft-edge/webview2/get-started/winui2.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ ms.date: 02/21/2023
 # Get started with WebView2 in WinUI 2 (UWP) apps
 
 In this tutorial, you:
-*  Set up your development tools for creating UWP apps that use WebView2 to display web content.
+*  Set up your development tools for creating UWP apps that use WebView2 to display web content. These instructions are also applicable to Xbox. 
 *  Create an initial WinUI 2 (UWP) app.
 *  Install the **Microsoft.UI.Xaml** package (WinUI 2) for the project.
 *  Add a WebView2 control that displays webpage content.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/get-started/winui2.md:17]: probably above the paragraph:
WinUI 2 supports UWP only. These controls are backward-compatible.
add new paragraph/sentence about platforms/scope applicability:

This article applies to Windows and Xbox.

File: microsoft-edge/webview2/roadmap.md [hunk 5]
```diff
@@ -69,6 +69,12 @@ You can access WebView2 controls in your applications using [Windows UI Library
 XAML Island support requires additional work and may be considered for future releases.
 
 
+<!-- ------------------------------ -->
+#### Xbox
+
+WebView2 is now available on Xbox devices.
```
Comment (captainbrosset) [inline microsoft-edge/webview2/roadmap.md:75]:
```suggestion
WebView2 is available on Xbox devices. 
```

References to time-based things, like the word "now", are not usually needed and often force us to update docs after a while. I suggest just removing it.

File: microsoft-edge/webview2/index.md (hunk 1 above)
Comment (nishitha-burman) [reply microsoft-edge/webview2/index.md:91]: I like the second option: WebView2 is also supported on the following devices:

Comment (mikehoffms) [reply microsoft-edge/webview2/index.md:91]: @nishitha-burman, I can make such changes locally in VS Code and then push commits to your PR, soon.

File: microsoft-edge/webview2/roadmap.md (hunk 5 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/roadmap.md:75]: keeping "now"

File: microsoft-edge/webview2/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/index.md:91]: WebView2 is also supported on the following devices:

File: microsoft-edge/webview2/get-started/winui2.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/get-started/winui2.md:17]: Result:
```
In this tutorial, you:
*  Set up your development tools for creating UWP apps that use WebView2 to display web content.
*  Create an initial WinUI 2 (UWP) app.
...

<!-- ------------------------------ -->
#### Platforms

This article applies to Windows and Xbox.

WinUI 2 supports UWP only.  These controls are backward-compatible.
```

File: microsoft-edge/webview2/get-started/winui2.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/get-started/winui2.md:14]: Result:
```
In this tutorial, you:
*  Set up your development tools for creating UWP apps that use WebView2 to display web content.
*  Create an initial WinUI 2 (UWP) app.
...

<!-- ------------------------------ -->
#### Platforms

This article applies to Windows and Xbox.

WinUI 2 supports UWP only.  These controls are backward-compatible.
```

File: microsoft-edge/webview2/index.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/index.md:92]: omitted windows from list.

result:
```
WebView2 is also supported on the following devices:
*  Xbox
*  HoloLens 2
```

**Review by mikehoffms — APPROVED** [review]
Writer/Editor review 2

**Pull request merged**